tests/test_sensor.py

```
@@ -1015,6 +1087,11 @@ def collector():
 
 
 def _setup_entry_with_all_syncs(state: dict, make_config_entry):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_setup_entry_with_all_syncs`

(ANN202)